components/header/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './index.css'
+import Logo from '../logo'
+import GlobalHeader from '../../design/header'
+import Avatar from '../../design/avatar'
+import SearchBar from '../searchbar'
+
+const Header = React.memo(({ className }) => {
+  return (
+    <GlobalHeader className={className}>
```

Comment:
I would call it `AppBar` just because it's more meaningful and common for Material Design. Also, the opposite works — AppBar should not always be a header and vice versa.